charms/kfp-ui/requirements-integration.txt

```diff
@@ -68,56 +66,60 @@ hyperframe==6.0.1
     # via
     #   h2
     #   selenium-wire
-idna==2.10
+idna==3.4
     # via
     #   -r ./requirements.txt
     #   requests
     #   trio
-iniconfig==1.1.1
+importlib-resources==5.12.0
+    # via
+    #   -r ./requirements.txt
+    #   jsonschema
+iniconfig==2.0.0
     # via pytest
-ipdb==0.13.9
+ipdb==0.13.13
     # via pytest-operator
-ipython==8.7.0
+ipython==8.12.2
     # via ipdb
 jedi==0.18.2
     # via ipython
 jinja2==3.1.2
     # via pytest-operator
-jsonschema==3.2.0
+jsonschema==4.17.3
     # via
     #   -r ./requirements.txt
     #   serialized-data-interface
-juju==3.0.4
+juju==2.9.42.4
     # via
     #   -r ./requirements-integration.in
     #   pytest-operator
 jujubundlelib==0.5.7
     # via theblues
 kaitaistruct==0.10
     # via selenium-wire
-kubernetes==17.17.0
+kubernetes==26.1.0
     # via juju
 macaroonbakery==1.3.1
     # via
     #   juju
     #   theblues
-markupsafe==2.1.1
+markupsafe==2.1.3
     # via jinja2
 matplotlib-inline==0.1.6
     # via ipython
-mypy-extensions==0.4.3
+mypy-extensions==1.0.0
     # via typing-inspect
 oauthlib==3.2.2
     # via requests-oauthlib
 oci-image==1.0.0
     # via -r ./requirements.txt
-ops==1.5.4
```

Comment:
that is your guy causing issues

Reply:
@i-chvets https://github.com/canonical/kfp-operators/pull/224#discussion_r1238416540

Reply:
> that is your guy causing issues

I know, but since I'm updating requirements anyway, let's just go all the way.

I updated the code and unit tests in the latest commits to reflect changes in SDI and now unit tests work.